torch/csrc/jit/codegen/cuda/lower_expr_sort.cpp

```diff
@@ -0,0 +1,366 @@
+#include <torch/csrc/jit/codegen/cuda/lower_expr_sort.h>
+
+#include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
+#include <torch/csrc/jit/codegen/cuda/ir_utils.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+namespace {
+
+TensorView* findOutputTensor(Expr* expr) {
```

Comment:
since these are local helpers (so no separate declarations), please add brief function comments for each

Reply:
Comment added